scripts/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+  echo -e "${INF} cleaning up"
+
```

Comment:
`ERR`, `INF` etc only defined in `init.sh` -- when should that be called? 
in any case -- better to split them out so that they can be `source`d in the called script.